tests/unit/components/Results/KeywordTooltip.spec.js

```diff
@@ -0,0 +1,24 @@
+import {shallowMount} from "@vue/test-utils";
+import KeywordTooltip from "@/components/Results/KeywordTooltip.vue"
+
+describe('KeywordTooltip.vue', () => {
+  let wrapper;
+  let keyword = {
+    label: 'ABC',
+    synonyms: ["abc", "def:test"],
+    definitions: ["first def"],
+    expandedNames: []
+  };
+
+  it("can be mounted", () => {
+    wrapper = shallowMount(KeywordTooltip, { propsData: { keyword: keyword } });
+    expect(wrapper.name()).toMatch("KeywordTooltip");
```

Comment:
Please use ```wrapper.vm.$options.name``` instead of ```wrapper.name()``` as later one is deprecated and it is giving console error while running test